feat: nicer display for fbc pipelines

ralphbean opened the pull request:
Functionally, this was all doing the right thing before. Tasks later in the pipeline wait automatically for preceding tasks if the output of a preceding task is used as a param of a later task.

However, our UI doesn't render the pipeline nicely if we only rely on those dependencies. By using `runAfter`, we can hint to the UI how the pipelinerun should be rendered.

happybhati reviewed (changes requested):
Small addition
Bump up pipeline [version](https://github.com/konflux-ci/release-service-catalog/pull/551/files#diff-3778095b07e9be9d621f33c7b73cd675da51308dd923e44c58458738d8c023cdL7) and add README-changelog entry for the updated version.

And thanks for the PR 🫡

ralphbean commented:
@happybhati done, updated!

happybhati commented:
> @happybhati done, updated!

I think you updated **task** version and readme instead of **pipeline**.

mmalina commented on tasks/update-ocp-tag/README.md:
Agreed with Happy, this is the wrong README and wrong place to bump the version - those changes should be in the pipeline.

BTW, we had this exact issue before and added some explicit `runAfter` values in other pipelines. But not this one I guess :)

johnbieren replied:
@davidmogar did you ever open a bug against openshift pipelines for this?

davidmogar replied:
Never offered to do it ;)

johnbieren replied:
I will create you a critical Jira to open it :wink:

ralphbean commented:
> > @happybhati done, updated!
> 
> I think you updated **task** version and readme instead of **pipeline**.

:cry: oh my. Fixing!

ralphbean committed:
feat: nicer display for fbc pipelines

Functionally, this was all doing the right thing before. Tasks later in
the pipeline wait automatically for preceding tasks if the output of a
preceding task is used as a param of a later task.

However, our UI doesn't render the pipeline nicely if we only rely on
those dependencies. By using `runAfter`, we can hint to the UI how the
pipelinerun should be rendered.

Signed-off-by: Ralph Bean <rbean@redhat.com>

ralphbean commented:
OK, let's try that again.
